src/main/java/com/dougnoel/sentinel/elements/PageElement.java

```diff
@@ -267,6 +272,65 @@ public PageElement click() {
 		}
 		return this;
 	}
+	
+	/**
+	 * Drags the current element on top of the target element.
+	 * @param target PageElement the element the target is being dragged and dropped onto 
+	 * @return PageElement (for chaining)
+	 */
+	public PageElement dragAndDrop(PageElement target) {
+		new Actions(driver).dragAndDrop(this.toWebElement(), target.toWebElement()).build().perform();
+		return this;
+	}
+	
+	/**
+	 * Drags the current element on top of the target element via executing the drag and drop helper.js file
+	 * @param target PageElement the element the target is being dragged and dropped onto 
+	 * @return PageElement (for chaining)
+	 */
+	
+	public PageElement dragAndDropToTarget(PageElement target) {
+		dragAndDropToTarget(this.toWebElement(), target.toWebElement());
+		return this;
+	}
+	
+	/**
+	 * Reads the file from a given path.
+	 * @param path
+	 * @return 
+	 */
+	
+	public static String readFile(String path, Charset encoding)
+	{ 
+
+		byte[] encoded = null;
+		try {
+			encoded = Files.readAllBytes(Paths.get(path));
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}
+	    return new String(encoded, encoding);
+	}
+	
+	/**
+	 * Executes the script byt calling executeDrageAndDrop() function on a source element passing the target element as a dropTarget
+	 * @param source,target 
+	 * @return PageElement (for chaining)
+	 */
+	
+	  public PageElement dragAndDropToTarget(WebElement source, WebElement target) {		  
```

Comment:
This code should be added to the above method as mentioned. That method has better comments.